### PR TITLE
Form css

### PR DESCRIPTION
**Pull request by SousaLuis**

### CODE REVIEW

File: templates/user/login.html.twig [hunk 1]
```diff
@@ -5,23 +5,23 @@
...
                 </div>
-                <div class="login-row">
+                <div>
                     <label for="password">Mot de passe:</label>
-                    <input type="password" id="password" name="_password"/>
+                    <input type="password" id="password" name="_password" placeholder="Mot de passe"/>
                 </div>
-                {# If you want to control the URL the user is redirected to on success #}
-                <input type="hidden" name="_target_path" value="{{ path('vehicle_index') }}"/>
+            </div>
```
Comment (rocambille) [inline templates/user/login.html.twig:25]: this end tag closes opening tag in line 12, while the form is still opened. your opening/closing tags don't match

**Review by rocambille — CHANGES_REQUESTED** [review]

File: templates/user/login.html.twig (hunk 1 above)
Comment (JmJames51) [reply templates/user/login.html.twig:25]: fixed

Comment (rocambille) [reply templates/user/login.html.twig:25]: count opening/closing div tags

Comment (JmJames51) [reply templates/user/login.html.twig:25]: yep, i've missed a end tag, should be fix now

**Pull request merged**